### DISCUSSION TITLE
Updating issue labels?

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#1323] [(A) Anything]
Maybe it would be nice to update the issue labels to sort things better. I have a few ideas for issue categories, but it'd be nice to get some feedback.

- Priorities (low/medium/high/critical)
- Type (refactor/feature req/patchfix)
- Status (invalid/in progress/fixed)

What does everyone else think? Maybe we can also add some snazzy colors.

**Comment by donaldli2020** [comment]
<details>
<summary>Quote Reply</summary>
Maybe it would be nice to update the issue labels to sort things better. I have a few ideas for issue categories, but it'd be nice to get some feedback.<br><br>
* Priorities (low/medium/high/critical)<br>
* Type (refactor/feature req/patchfix)<br>
* Status (invalid/in progress/fixed)<br><br>
* What does everyone else think? Maybe we can also add some snazzy colors.
</details>

Sounds cool to me!

**Comment by SakuraKitsunee** [comment]
I think it's a pretty good idea, especially because it's been the same FOREVER!!!!

**Reply by afkvido** [reply]
ur collaborator?

**Reply by SakuraKitsunee** [reply]
@afkvido yeahhhh...

**Reply by afkvido** [reply]
lmao at first i thought u were a random who made the prodigy account ordering thing

**Reply by PatheticMustan** [reply]
@SakuraKitsunee you've been removed as an outside collaborator and invited to prodigy-hacking/triage, please accept

**Reply by afkvido** [reply]
LMAO **bruh**
